tests/amr/data/particles/stream_pack/test_main.cpp

```diff
@@ -160,36 +160,36 @@ TYPED_TEST(StreamPackTest, ShiftTheiCellWhenPackStreamWithPeriodics)
 
 
 
-TYPED_TEST(StreamPackTest, PackInTheCorrectBufferWithPeriodics)
-{
-    using ParticlesData = TypeParam;
-    constexpr auto dim  = ParticlesData::dimension;
-
-    ParticlesData param;
-    auto& particle    = param.particle;
-    auto& sourceData  = param.sourceData;
-    auto& cellOverlap = param.cellOverlap;
-    auto& destData    = param.destData;
-
-    particle.iCell = ConstArray<int, dim>(16);
-
-    sourceData.patchGhostParticles.push_back(particle);
-
-    SAMRAI::tbox::MessageStream particlesWriteStream;
-
-    sourceData.packStream(particlesWriteStream, *cellOverlap);
-
-    SAMRAI::tbox::MessageStream particlesReadStream{particlesWriteStream.getCurrentSize(),
-                                                    SAMRAI::tbox::MessageStream::Read,
-                                                    particlesWriteStream.getBufferStart()};
-
-    destData.unpackStream(particlesReadStream, *cellOverlap);
-
-    auto expectediCell = ConstArray<int, dim>(0);
-
-    ASSERT_THAT(destData.domainParticles.size(), Eq(1));
-    ASSERT_THAT(destData.domainParticles[0].iCell, Eq(expectediCell));
-}
+// TYPED_TEST(StreamPackTest, PackInTheCorrectBufferWithPeriodics)
```

Comment:
forgot to update or we don't need it?

Reply:
it's back in, modified slightly. Was indeed removed because it puts a particle in a ghost cell and expects it in the neighbor domain.
Changed it to put the particle in the last domain cell of source, and expect it in ghost of destination in cell -1 (periodicity).
Seems to work.